_config.dev.yml

```diff
@@ -15,6 +15,7 @@ exclude:
   - Gemfile.lock
   - package.json
   - node_modules
+  - src
```

Comment:
Since it's not ignoring `assets/build`, this ensures Jekyll waits to regenerate the site until after Webpack finishes and outputs the bundles.